cmd/mt-index-cat/out/out.go

```diff
@@ -45,7 +44,7 @@ func Template(format string) func(d schema.MetricDefinition) {
 
 	tpl := template.Must(template.New("format").Funcs(funcs).Parse(format))
 
-	return func(d schema.MetricDefinition) {
+	return func(d idx.MetricDefinition) {
```

Comment:
trying to print properties from a idx.MetricDefinition is probably going to print out the raw uintptrs etc

Reply:
maybe the answer is to convert to schema.MetricDefinition before feeding to the template?